internal/connectors/engine/workflow/delete_bank_bridge_connection_data_test.go

```diff
@@ -13,73 +12,19 @@ import (
 
 func (s *UnitTestSuite) Test_DeleteBankBridgeConnectionData_FromConnectionID_Success() {
 	connectionID := "test-connection-id"
+	connectorID := models.ConnectorID{
+		Reference: uuid.New(),
+		Provider:  "test",
+	}
 	psuID := uuid.New()
 
-	// Mock payments list with connection ID filter
-	s.env.OnActivity(activities.StoragePaymentsListActivity, mock.Anything, mock.Anything).Once().Return(
-		&bunpaginate.Cursor[models.Payment]{
-			PageSize: 2,
-			HasMore:  false,
-			Data: []models.Payment{
-				{
-					ConnectorID: s.connectorID,
-					ID:          s.paymentPayoutID,
-					Metadata: map[string]string{
-						models.ObjectConnectionIDMetadataKey: connectionID,
-					},
-				},
-				{
-					ConnectorID: s.connectorID,
-					ID: models.PaymentID{
-						PaymentReference: models.PaymentReference{
-							Reference: "test-2",
-							Type:      models.PAYMENT_TYPE_PAYOUT,
-						},
-						ConnectorID: s.connectorID,
-					},
-					Metadata: map[string]string{
-						models.ObjectConnectionIDMetadataKey: connectionID,
-					},
-				},
-			},
-		},
-		nil,
-	)
-
-	// Mock payment deletions
-	s.env.OnActivity(activities.StoragePaymentsDeleteActivity, mock.Anything, s.paymentPayoutID).Once().Return(nil)
-	s.env.OnActivity(activities.StoragePaymentsDeleteActivity, mock.Anything, models.PaymentID{
-		PaymentReference: models.PaymentReference{
-			Reference: "test-2",
-			Type:      models.PAYMENT_TYPE_PAYOUT,
-		},
-		ConnectorID: s.connectorID,
-	}).Once().Return(nil)
-
-	// Mock accounts list with connection ID filter
-	s.env.OnActivity(activities.StorageAccountsListActivity, mock.Anything, mock.Anything).Once().Return(
-		&bunpaginate.Cursor[models.Account]{
-			PageSize: 1,
-			HasMore:  false,
-			Data: []models.Account{
-				{
-					ConnectorID: s.connectorID,
-					ID:          s.accountID,
-					Metadata: map[string]string{
-						models.ObjectConnectionIDMetadataKey: connectionID,
-					},
-				},
-			},
-		},
-		nil,
-	)
-
-	// Mock account deletion
-	s.env.OnActivity(activities.StorageAccountsDeleteActivity, mock.Anything, s.accountID).Once().Return(nil)
+	s.env.OnActivity(activities.StoragePaymentsDeleteFromConnectionIDActivity, mock.Anything, psuID, connectorID, connectionID).Once().Return(nil)
+	s.env.OnActivity(activities.StorageAccountsDeleteFromConnectionIDActivity, mock.Anything, psuID, connectorID, connectionID).Once().Return(nil)
```

Comment:
So if the list activities now return nil cursors, we no longer actually perform the account deletion activity (as you can see with line 78 being removed). So I don't think this test is testing the same path it was before your change?